### PR TITLE
Test: desktop with electrum custom backend running in regtest

### LINKED ISSUE #6244: Integration test for desktop with electrum backend
Implement test for suite-desktop app that achieves successful discovery with electrum backend running in regtest so it can be consistently run in CI.

### PR DESCRIPTION
**Pull request by karliatto**
## Description

Creates automatic integration test for suite desktop app with electrum custom backend running with a regtest bitcoind. This makes sure the communication between suite and electrum works properly since it checks that discovery happens successfully.

The tests were successfully passing in job https://gitlab.com/satoshilabs/trezor/trezor-suite/-/jobs/3020244787

## Related Issue

Resolve https://github.com/trezor/trezor-suite/issues/6244

### CODE REVIEW

File: docker/docker-compose.suite-desktop-ci.yml [hunk 1]
```diff
@@ -14,3 +15,9 @@ services:
     command: bash -c "yarn workspace @trezor/integration-tests test:desktop"
     volumes:
       - ../:/trezor-suite
+
+  electrum-regtest:
+    image: ghcr.io/vdovhanych/electrs:latest
```
Comment (vdovhanych) [inline docker/docker-compose.suite-desktop-ci.yml]: Use this image instead, please - `ghcr.io/trezor/electrs:latest`

File: docker/docker-compose.suite-desktop-test.yml [hunk 2]
```diff
@@ -20,3 +20,9 @@ services:
     volumes:
       - ../:/trezor-suite
       - /tmp/.X11-unix:/tmp/.X11-unix:rw
+
+  electrum-regtest:
+    image: ghcr.io/vdovhanych/electrs:latest
```
Comment (vdovhanych) [inline docker/docker-compose.suite-desktop-test.yml]: Use this image instead, please - `ghcr.io/trezor/electrs:latest`

**Review by vdovhanych — CHANGES_REQUESTED** [review]

File: docker/docker-compose.suite-desktop-ci.yml (hunk 1 above)
Comment (karliatto) [reply docker/docker-compose.suite-desktop-ci.yml]: [ff2ee66](https://github.com/trezor/trezor-suite/pull/6258/commits/ff2ee66c35ff8a04f1f0477b2b4581aa1347f8f7)

File: docker/docker-compose.suite-desktop-test.yml (hunk 2 above)
Comment (karliatto) [reply docker/docker-compose.suite-desktop-test.yml]: https://github.com/trezor/trezor-suite/pull/6258/commits/d3e232a4ebd5775b01788dc0cde5d7e71475e5dc

**Review by vdovhanych — APPROVED** [review]

File: packages/integration-tests/controllerManager.js [hunk 3]
```diff
@@ -0,0 +1,134 @@
+module.exports = (controller, firmwares) => ({
```
Comment (mroz22) [inline packages/integration-tests/controllerManager.js:1]: nice, sort of future trezor-user-env-link API here. could you please somehow incorporate this file into `packages/trezor-user-env-link` ? Or maybe we could do  this later, no problem

**Review by mroz22 — APPROVED** [review]
nice, looking forward future improvements

**Comment by karliatto** [comment]
There is follow up to move suite-desktop test to suite-desktop/e2e https://github.com/trezor/trezor-suite/issues/6305

**Pull request merged**